plugin/tests/binops.rs

```diff
@@ -1,128 +0,0 @@
-#![cfg_attr(test, feature(plugin))]
```

Comment:
This has been renamed to `integration.rs`. As tests are compiled and then ran, `target/mutagen/mutations.txt` is overwritten, so we can't have multiple test files.
I'm working on a branch that extends current `compiletest`-tests that would allow to have multiple test files.